backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -29,37 +31,68 @@ object WorkflowBackendActor {
   sealed trait AbortResponse extends WorkflowBackendActorResponse
   case class AbortSucceeded(jobKey: BackendJobDescriptorKey) extends AbortResponse
   case class AbortFailed(jobKey: BackendJobDescriptorKey, throwable: Throwable) extends AbortResponse
+
+  sealed trait InitializationResponse extends WorkflowBackendActorResponse
+  case object InitializationSuccess extends InitializationResponse
+  case class InitializationFailed(reason: Throwable) extends InitializationResponse
```

Comment:
I like keeping this non-final, implementers can throw classes of failures more specifically by subtype.  For JES in particular we can draw distinctions between "could not upload auth file" and "you're missing a docker runtime attribute in call 'foo'.".